bpf_assets/perf_event/perf_event.c

```diff
@@ -88,8 +88,12 @@ static void safe_array_add(u32 idx, u16 *array, u16 value)
 
 int sched_switch(switch_args *ctx)
 {
-    u64 pid = bpf_get_current_pid_tgid() & 0xffffffff;
+    u64 pid = bpf_get_current_pid_tgid();
```

Comment:
the pid is in the [low 32 bits](https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md#4-bpf_get_current_pid_tgid)

Reply:
The `bpf_get_current_pid_tgid() & 0xffffffff` was not giving to me the right pid of the process. 
Then I saw other eBPF examples, like this one [here](https://github.com/iovisor/bcc/blob/master/tools/cpudist.py) that are not using a mask to get the pid. 
After many tests, using it without the mask, the command was giving the right pid of the process.